src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -332,15 +333,8 @@ void TorrentHandleImpl::setAutoManaged(const bool enable)
 
 QVector<TrackerEntry> TorrentHandleImpl::trackers() const
 {
-    const std::vector<lt::announce_entry> nativeTrackers = m_nativeHandle.trackers();
-
-    QVector<TrackerEntry> entries;
-    entries.reserve(nativeTrackers.size());
-
-    for (const lt::announce_entry &tracker : nativeTrackers)
-        entries << tracker;
-
-    return entries;
+    const std::vector<lt::announce_entry> &nativeTrackers = m_nativeHandle.trackers();
```

Comment:
you can't store a reference like this.

Reply:
I can. Reference extend temporary lifetime.